docs/diagrams/SystemExceptionEnum.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
```

Comment:
Since this is not part of the code and people refer to the diagram, not so much of this code, so I think can remove all the whitespace (extra lines) since it is inside the enum.